### DISCUSSION TITLE
无法扫码登录

### DISCUSSION THREAD

**Discussion by birdding** [Mrs4s/go-cqhttp#1477] [Q&A]
请问，登录提示需要扫二维码，但是扫码提示“当前登录环境异常，为保障您的账号安全，暂时无法登录。建议将两个设备连接同一个网络或WIFI后重新扫码”，这个问题如何处理呢？qq安全设置里面登录保护已经关闭了。

**Comment by Ink-33** [comment]
duplicated #1469

**Reply by birdding** [reply]
谢谢回复，但我将自己电脑上登录成功的device.json复制到远程服务器上，运行后还是提示这个报错

**Reply by liyk-master** [reply]
重复尝试
